src/main/java/frc/robot/commands/drive/Drive.java

```diff
@@ -26,6 +23,34 @@ public Drive() {
         requires(Drivetrain.getInstance());
     }
 
+    public static double powerCurve(double x) {
```

Comment:
Still `public`, which is kinda sketch. But at least it's a member of the right class now!